angr_platforms/risc_v/arch_riscv.py

```diff
@@ -157,7 +157,7 @@ def __init__(self, endness=Endness.LE):
                  general_purpose=True, vex_offset=120),
         Register(name='x31', size=4, alias_names=('t6',),
                  general_purpose=True, vex_offset=124),
-        Register(name='ip', alias_names={'pc', }, size=4, vex_offset=128),
+        Register(name='ip', alias_names={'pc', 'ip_at_syscall',}, size=4, vex_offset=128),
```

Comment:
Only quibble about this PR: ip_at_syscall is supposed to be its own register - the way syscalls work in angr is that there is a special "kernel region" set up by the SimOS, and when a syscall happens the ip is set to somewhere in that region - this helps with static analysis - and ip_at_syscall is used as the address to return to after the syscall finishes.